docs/conf.py

```diff
@@ -102,6 +107,8 @@ def check_dot_command():
     #    of the inmanta-core documentation. During the ISO docs build,
     #    this value will be overwritten with the ISO product version.
     version = "1.0.0"
+    iso_major = int(version.split(".")[0])
```

Comment:
If we name this `version_major` we can do this unconditionally (we'd still use it conditionally though) so that it's always here and we can drop the PR on the iso repo.

Reply:
I don't fully get what you mean here.

Reply:
I added a `version_major` variable that is set when performing an iso build (and used in iso-only parts of the docs).